types/types.go

```diff
@@ -58,7 +58,7 @@ type Image interface {
 	IntendedDockerReference() string
 	// Manifest is like ImageSource.GetManifest, but the result is cached; it is OK to call this however often you need.
 	// NOTE: It is essential for signature verification that Manifest returns the manifest from which LayerDigests is computed.
-	Manifest() ([]byte, error)
+	Manifest() ([]byte, string, error)
```

Comment:
Ah, true. Then perhaps `getParsedManifest` should add a `case ""`, both to be explicit that this can happen and to have a better error message.